nonmoodlefeedback.php

```diff
@@ -34,9 +34,9 @@
 $instance = optional_param('instance', 0, PARAM_INT);
 
 $report = new report_myfeedback\local\report();
-$report->init();
-$report->setup_external_db();
-if (!empty($feedname) && $gradeid2 && $userid2) {
+//$report->init();
+//$report->setup_external_db();
```

Comment:
if this is not needed, don't just comment it out, remove it

Reply:
removed